interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -276,7 +277,7 @@ protected ManagedChannel createChannel() {
         if (useTestCa) {
           try {
             sslContext = GrpcSslContexts.forClient().trustManager(
-                    TestUtils.loadCert("ca.pem")).build();
+                TestUtils.loadCert("ca.pem")).sslProvider(SslProvider.OPENSSL).build();
```

Comment:
The change here and in `TestServiceServer` isn't what we want. First, the change here only applies when using the test CA. But also, these aren't run as part of the build (only in the interop testing environment).